driver-core/src/main/com/mongodb/client/model/expressions/EntryExpression.java

```diff
@@ -25,12 +28,15 @@
  * {@linkplain Expression value}. Entries are used with
  * {@linkplain MapExpression maps}.
  *
- * Entries are {@linkplain Expression#isDocumentOr document-like} and
+ * <p>Entries are {@linkplain Expression#isDocumentOr document-like} and
  * {@linkplain Expression#isMapOr map-like}, unless the method returning the
  * entry specifies otherwise.
```

Comment:
[optional]

The closing tag is missing. You will find many places in the existing code where this is the case, but at some point we had a discussion (in a PR) were it was decided that we should specify closing tags. See `Locks` for an example of a new code that does this.

I only now noticed this, and after checking, saw that this is how all docs in the expressions API are written. Marking this as optional to avoid blocking the progress.